pkg/vault/vault.go

```diff
@@ -179,7 +179,7 @@ func (vs *Service) Decrypt(appName, ciphertext string) (string, error) {
 }
```

Comment:
Let's move this into pkg/storage/vault

Reply:
As I checked, the opx import existing vault pkg https://github.com/openware/opx/blob/5bef775f55732eb25dbc1823c0fcb19098932c49/secrets/secrets.go#L8